TradeBeacons/data/scripts/entity/tradebeacon.lua

```diff
@@ -140,7 +140,7 @@ function TradeBeacon.updateServerCharged()
     TradeBeacon.registerWithPlayer()
 
     if traderAffinity ~= nil and traderAffinity > 0 and random():getFloat() < traderAffinity then
-        Sector():addScriptOnce("data/scripts/player/spawntravellingmerchant.lua")
+        Sector():addScriptOnce("data/scripts/player/events/spawntravellingmerchant.lua")

```

Comment:
I saw this bad path and fixed it. this might require more balancing since there will be more travelling merchants.

The latest update (0.27) conveniently spits out a message when you do this now. I don't think it used to do that.